packages/myst-frontmatter/src/settings/validators.ts

```diff
@@ -0,0 +1,67 @@
+import type { ValidationOptions } from 'simple-validators';
+import { defined, incrementOptions, validateChoice, validateObjectKeys } from 'simple-validators';
+import type { ProjectSettings } from './types.js';
+import { validateMystToTexSettings } from './validatorsMystToTex.js';
+
+const OUTPUT_REMOVAL_OPTIONS: Required<ProjectSettings>['output_stderr'][] = [
+  'show',
+  'remove',
+  'remove-warn',
+  'remove-error',
+  'warn',
+  'error',
+];
+export const PROJECT_SETTINGS = [
+  'output_stderr',
+  'output_stdout',
+  'output_matplotlib_strings',
+  'myst_to_tex',
+];
+export const PROJECT_SETTINGS_ALIAS = {
+  stderr_output: 'output_stderr',
+  stdout_output: 'output_stdout',
+  mystToTex: 'myst_to_tex',
```

Comment:
This could just be `tex` settings? Maybe we add that as an alias at least? Unless there's some other meaning for "tex settings" besides tex exports.

Reply:
I have been wanting some settings for the texToMyst parser for a while, so that was leaving room for that. Will add the alias!